### PR TITLE
Better stream error

### PR DESCRIPTION
**Pull request by windmemory**
Related to https://github.com/wechaty/wechaty-puppet-hostie/issues/93

### CODE REVIEW

File: src/stream/typed-stream.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import {
```
Comment (huan) [inline src/stream/typed-stream.ts:1]: I believe this file had been deleted by me days ago.

**Review by huan — CHANGES_REQUESTED** [review]
Thank you very much for this great improvement with streaming with error handlers!

Please remove the `test.dat` and check the file `typed-stream.ts`, then I believe it will be ready to merge.

**Comment by windmemory** [comment]
Updated according to comments :P

**Review by huan — APPROVED** [review]
LGTM

**Comment by huan** [comment]
Brilliant changes! merged.

**Comment by windmemory** [comment]
> Brilliant changes! merged.

Thanks for your help on the PR!

**Pull request merged**